commands/commands.go

```diff
@@ -85,7 +85,7 @@ func FullError(err error) {
 
 func errorWith(err error, fatalErrFn func(error, string, ...interface{}), errFn func(string, ...interface{})) {
 	if Debugging || errors.IsFatalError(err) {
-		fatalErrFn(err, "")
+		fatalErrFn(err, "%s", err)
```

Comment:
I was wondering about this when I wrote this patch. My thinking is to leave it as-is (`"%s", err.Error()`), since that's what we're doing with [the errFn call](https://github.com/github/git-lfs/blob/7ccc6b5cec1f8eec1d05f8e582c8e4b825d3bcc2/commands/commands.go#L92).

I like that giving a blank format string means no output is generated (other than the standard panic message about logging to a file) and that that behavior is explicit.

I'm definitely open to changing this, though I think we'd want to do it at both call sites.